static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -187,6 +189,10 @@ define([
             blockEngagementBanner: options.blockEngagementBanner || false,
             engagementBannerParams: options.engagementBannerParams || {},
             isOutbrainCompliant: options.isOutbrainCompliant || false,
+
+            // Set useLocalViewLog to true if only the views for the respective test
+            // should be used to determine variant viewability
+            useLocalViewLog: options.useLocalViewLog || false,
```

Comment:
Yeah - me too. I was see-sawing back and forth on this. Currently thinking it should go at the test level, so happy to change?